src/pages/main.js

```diff
@@ -51,10 +42,6 @@ const Main = class Main extends Component {
           <Fragment>
             <Fragment>
               <Display />
-              <Legend />
```

Comment:
The legend shouldn't have been removed here. Also, it'll need to be restyled / repositioned to float to the right of the side panel when it's present.